### PR TITLE
madspin cards updated

### PR DESCRIPTION
**Pull request by wajidalikhan**
Dear GEN colleagues, 

Removed the redundant / useless w decay information present in the madspin cards:   
decay t > w+ b, w+ > all all
decay t~ > w- b~, w- > all all
decay w+ > all all
decay w- > all all

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-2500toInf_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ T        = pdfwgt           ! for ickkw=1, perform pdf reweighting
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-2500toInf_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM_run_card.dat:71]: https://twiki.cern.ch/twiki/bin/view/CMS/MCKnownIssues#pdfwgt_setting
pdfwgt = T should modelling problems.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-2500toInf_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+ -1  = dralmax ! maxdistance between gamma and lepton
+#*********************************************************************
+# Minimum and maximum invariant mass for pairs                       *
+# WARNING: for four lepton final state mmll cut require to have      *
+#          different lepton masses for each flavor!                  *           
+#*********************************************************************
+ 0   = mmjj    ! min invariant mass of a jet pair 
+ 0   = mmbb    ! min invariant mass of a b pair 
+ 0   = mmaa    ! min invariant mass of gamma gamma pair
+50   = mmll    ! min invariant mass of l+l- (same flavour) lepton pair
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-2500toInf_LO_MLM/tt0123j_5f_ckm_HT-2500toInf_LO_MLM_run_card.dat:174]: as this is not relevant i would set it to 0.

**Comment by wajidalikhan** [comment]
Thanks @agrohsje, modified the card with madspin fix and pdfwgt=F and mmll = 0. Hopefully this should resolve most of the things.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-1200to2500_LO_MLM/tt0123j_5f_ckm_HT-1200to2500_LO_MLM_customizecards.dat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card mass 25 125.0
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_5f_ckm_HT-1200to2500_LO_MLM/tt0123j_5f_ckm_HT-1200to2500_LO_MLM_customizecards.dat:3]: the process is pure qcd, so no qed lines including the higgs. you can remove the line. on the other hand you change the top mass, so you might want to change the top width with auto too. will be done anway by madspin, but just to make the cards cleaner.

**Comment by agrohsje** [comment]
hi wajid. as we are already working on this, i made proposals to clean up customize card and madspin. maybe you can quickly do that. @sihyunjeon please check and merge once it is cleaned up by @wajidalikhan .
thanks so much !

**Comment by wajidalikhan** [comment]
Hi @agrohsje, thank you very much for your suggestion and your time. I totally agree in cleaning the useless / unnecessary stuff from the data cards. The cleaner we make the system the less error we commit in the MC that we produce. I will remove the Higgs mass from the customizations, 
>>> set param_card mass 25 125.0

or if we have by default the top mass set to 172.5 we can also remove:
>>> set param_card mass 6 172.5

**Review by wajidalikhan — COMMENTED** [review]
Agreed!

**Comment by agrohsje** [comment]
I don't think 172.5 is the default. So better to have it.

**Comment by agrohsje** [comment]
You were a bit too ambitious with the clean-up. ell and q still needs to be defined. They are not part of the model.

**Comment by wajidalikhan** [comment]
I was a bit too fast :D defined them again!

**Comment by wajidalikhan** [comment]
Hi @sihyunjeon, @agrohsje  shall we close the PR it if everything is fine.

**Comment by agrohsje** [comment]
Hi @sihyun. Feel free to merge. Thanks!

**Pull request merged**